plugin/src/main/java/io/snyk/languageserver/protocolextension/SnykExtendedLanguageClient.java

```diff
@@ -118,12 +121,8 @@ public void addTrustedPaths(SnykTrustedFoldersParams param) {
     var trustedPaths = storedTrustedPaths.split(File.pathSeparator);
     var pathSet = new HashSet<>(Arrays.asList(trustedPaths));
     pathSet.addAll(Arrays.asList(param.getTrustedFolders()));
-    Preferences.getInstance().store(Preferences.TRUSTED_FOLDERS,
-        pathSet.stream()
-        .filter(s -> !s.isBlank())
-        .map(s -> s.trim())
-        .distinct()
-        .collect(Collectors.joining(File.pathSeparator)));
+    Preferences.getInstance().store(Preferences.TRUSTED_FOLDERS, pathSet.stream().filter(s -> !s.isBlank())
```

Comment:
Personally I prefer the copped version but don't worry about it

Reply:
me, too, but the auto-formatter didn't ask me :(